base_layer/wallet/src/output_manager_service/storage/sqlite_db/mod.rs

```diff
@@ -1332,18 +1332,19 @@ impl KnownOneSidedPaymentScriptSql {
         let script = known_script.script.to_bytes().to_vec();
```

Comment:
I cant leave a comment on the line above, 
But `known_script.private_key` is not zeroized, and `known_script` is moved here and dropped after this function.

Reply:
Very good observation. I refactored it to zeroize the associated sensitive data.